src/constants/api.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import { ENV } from "@paypal/sdk-constants/src";
+
+export const HEADERS = {
+  AUTHORIZATION: "authorization",
+  CONTENT_TYPE: "content-type",
+  PARTNER_ATTRIBUTION_ID: "paypal-partner-attribution-id",
+};
+
+export const ELIGIBLE_PAYMENT_METHODS = "v2/payments/find-eligible-methods";
+
+// $FlowFixMe
+export const BASE_API_URL = {
```

Comment:
do we get these from csnw? https://github.com/paypal/paypal-sdk-client/blob/main/src/global.js#L46

that might be how we get the stage urls